C:30:"PHPUnit\Runner\TestResultCache":1372:{a:2:{s:7:"defects";a:2:{s:120:"Tests\Core\Application\Service\GetUserBalanceUseCaseTest::testThatIfTheUserIsPresentThenAnSpecificResponseWillBeReturned";i:4;s:128:"Tests\Core\Domain\Service\IssueTransactionServiceTest::testThatIfTheAmountIsValidAndTheIssuerHasFundsThenThereWillBeATransaction";i:4;}s:5:"times";a:8:{s:120:"Tests\Core\Application\Service\GetUserBalanceUseCaseTest::testThatIfTheUserIsPresentThenAnSpecificResponseWillBeReturned";d:0.003;s:121:"Tests\Core\Application\Service\GetUserBalanceUseCaseTest::testThatIfNoUserHasBeenFountThenAnSpecificExceptionWillBeThrown";d:0.023;s:103:"Tests\Core\Application\Service\IssueTransactionUseCaseTest::testThatWithNoIssuerTheresAnExceptionThrown";d:0.008;s:105:"Tests\Core\Application\Service\IssueTransactionUseCaseTest::testThatWithNoRecieverTheresAnExceptionThrown";d:0.009;s:121:"Tests\Core\Application\Service\IssueTransactionUseCaseTest::testThatIfTwoUsersExistThereWillBeACallToTheUnderlyingService";d:0.009;s:109:"Tests\Core\Domain\Service\IssueTransactionServiceTest::testThatIfIssuerHasNotFundsThenAnExceptionWillBeThrown";d:0.003;s:111:"Tests\Core\Domain\Service\IssueTransactionServiceTest::testThatIfTheAmountIsNegativeThenAnExceptionWillBeThrown";d:0.003;s:128:"Tests\Core\Domain\Service\IssueTransactionServiceTest::testThatIfTheAmountIsValidAndTheIssuerHasFundsThenThereWillBeATransaction";d:0.003;}}}